package.json

```diff
@@ -18,12 +18,14 @@
   "author": "Continuation Labs <contact@continuation.io> (http://continuation.io/)",
   "license": "MIT",
   "dependencies": {
-    "bcrypt": "0.8.7"
+    "bcryptjs": "2.3.0"
   },
   "devDependencies": {
+    "aws-sdk": "2.6.1",
     "belly-button": "3.1.0",
```

Comment:
Since you're in here, can you change the deps to `major.x.x` for "known modules"